### ISSUE TITLE
Drupal 8 and Composer on Pantheon Without Continuous Integration Doc Update 

### ISSUE THREAD

**Issue by ElijahLynn** [pantheon-systems/documentation#3768]
Re: [Drupal 8 and Composer on Pantheon Without Continuous Integration](https://pantheon.io/docs/guides/drupal-8-composer-no-ci/)

Priority Medium?:

## Issue Description:
1. "Since we modified composer.json we will need to update Composer." at this point in the doc there haven't been any suggestions to modify composer.json
2. If we did modify composer.json, running `composer update` wouldn't update composer, it would pull in updates via composer.
3. `composer update` does not work at that point and fails with:
> find .circleci/scripts/pantheon/ -type f | xargs chmod 755
> find: `.circleci/scripts/pantheon/': No such file or directory
> chmod: missing operand after ‘755’
> Try 'chmod --help' for more information.
> Script find .circleci/scripts/pantheon/ -type f | xargs chmod 755 handling the post-update-cmd event returned with error code 123 

due to this script left behind:

> "post-update-cmd": [
>     "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
>     "find .circleci/scripts/pantheon/ -type f | xargs chmod 755",

## Suggested Resolution
Have someone go through that doc, step by step.

**Comment by rachelwhitton** [comment]
Thanks for reporting @ElijahLynn! 

@ataylorme can you take a look?

**Comment by ataylorme** [comment]
@ElijahLynn we have run through the doc step by step and many users have reported success. I was able to reproduce your error by 1) removing the `tests` directory and 2) not making any updates to `composer.json` before running `composer update`.

I think you missed the expanded section with `composer.json` modifications to the `scripts` section, which remove the `post-update-cmd`.

![d8-composer-no-ci-dropdown](https://user-images.githubusercontent.com/2133004/41241316-71478366-6d51-11e8-9454-a68e1811e771.png)

The entire section should be completed. I've opened #3772 to hopefully make that clear going forward.
